### ISSUE TITLE
Connection failure on Reward Calculator

### ISSUE THREAD

**Issue by jerkyrs** [icon-project/rewardcalculator#21]
This error appears to be from iconservice connecting to Rewards calculator

```
019-09-13 00:26:08,374 608 140513537681152 ERROR icon_inner_service.py(67) IconService commit_block message to RewardCalculator has timed-out (12)
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/iconservice/iiss/reward_calc/ipc/reward_calc_proxy.py", line 357, in commit_block
    response: 'CommitBlockResponse' = future.result(self.IPC_TIMEOUT_S)
  File "/usr/local/lib/python3.7/concurrent/futures/_base.py", line 434, in result
    raise TimeoutError()
concurrent.futures._base.TimeoutError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/iconservice/icon_inner_service.py", line 277, in _write_precommit_state
    self._icon_service_engine.commit(block_height, instant_block_hash, block_hash)
  File "/usr/local/lib/python3.7/site-packages/iconservice/icon_service_engine.py", line 1655, in commit
    context.engine.iiss.send_ipc(context, precommit_data)
  File "/usr/local/lib/python3.7/site-packages/iconservice/iiss/engine.py", line 687, in send_ipc
    self._reward_calc_proxy.commit_block(True, block_height, precommit_data.block.hash)
  File "/usr/local/lib/python3.7/site-packages/iconservice/iiss/reward_calc/ipc/reward_calc_proxy.py", line 360, in commit_block
    raise TimeoutException("commit_block message to RewardCalculator has timed-out")
iconservice.base.exception.TimeoutException: commit_block message to RewardCalculator has timed-out (12
```

**Comment by eunsoo-icon** [comment]
Let me know your running environment Info.
- How to run iconservice? Using docker or T-Bears or other
  - If you use docker, give me a docker image tag
  - If you use T-Bears, give me a T-Bears version
  - If you user other, give me a iconservice version and a icon_rc version
- iconservice version
  - pip show iconservice
- Reward calculator version
  - icon_rc -version

**Comment by jerkyrs** [comment]
Testnet3 with the following iconloop/prep-node:1909180304x565582

```
 _1909180304x565582@metanyx-icon-testnet :$ pip show iconservice
Name: iconservice
Version: 1.5.5
Summary: ICON Service for Python
Home-page: https://github.com/icon-project/icon-service
Author: ICON Foundation
Author-email: foo@icon.foundation
License: Apache License 2.0
Location: /usr/local/lib/python3.7/site-packages
Requires: setproctitle, secp256k1, msgpack, earlgrey, plyvel, iconcommons, iso3166
Required-by: tbears
 _1909180304x565582@metanyx-icon-testnet :$ icon_rc -version
icon_rc v1.1.0-10-g4bbe0fb, tags()-2019-09-17-18:04:53

```

**Comment by eunsoo-icon** [comment]
iconloop/prep-node:1909180304x565582 seems to have been released after registering this issue.

Please upload the iconservice.log and rc.log files for version and debugging information

**Comment by jerkyrs** [comment]
Please find attached
[logs.zip](https://github.com/icon-project/rewardcalculator/files/3634989/logs.zip)

**Comment by eunsoo-icon** [comment]
Error is occurred at 2019-09-13 00:26:08,374.

I can't find any useful information because your log files contain logs after 2019-09-17 22:45:27 and 2019/09/17 06:34:16.

ps. Our team find a similar timeout error case in v1.0.0 and fixed via #12 and applied to v1.1.0

**Comment by eunsoo-icon** [comment]
Fixed in v1.1.2
